codex/marketplace.md

```diff
@@ -0,0 +1,209 @@
+---
+title: CODEX-MARKETPLACE
+name: Codex Storage Marketplace
+status: raw
+tags: codex
+editor: Dmitriy <dryajov@status.im>
+contributors:
+---
+
+## Abstract
+
+This specification describes a method for Codex storage providers and client nodes to participate in a storage marketplace. 
+The goal is to create a storage marketplace that promotes durability.
+
+## Motivation
+The Codex network aims to create a peer-to-peer storage engine with strong data durability, 
+data persistence guarantees and node storage incentives.
+To reach this goal, a data availability and retrieval mechanism is needed.
+Support for light clients, like mobile devices should also be embraced.
+The protocol should remove complexity to allow for simple implementation and 
+simplify incentive mechanisms.
+
+## Semantics 
+
+The key words “MUST”, “MUST NOT”, “REQUIRED”, “SHALL”, “SHALL NOT”, “SHOULD”, “SHOULD NOT”, “RECOMMENDED”, “MAY”, and “OPTIONAL” in this document are to be interpreted as described in [2119](https://www.ietf.org/rfc/rfc2119.txt).
+
+### Definitions
+
+| Terminology  | Description |
+| --------------- | --------- |
+| storage providers | A Codex node that provides storage services to the marketplace. |
+| validator nodes | A Codex node that collects, validates and submits missing storage proofs for a reward. |
+| client nodes | The most common Codex node that interacts with other nodes to store, locate and retrieve data. Also considered to be an ephemeral node (light client). |
```

Comment:
An ephemeral node isn't a light client, in fact we avoid the usage of the term light client al together, due to it's ambiguity.